src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -105,6 +105,27 @@ export default [
         },
         getDescription: () => {},
     },
+    {
+        textTranslateKey: 'reportActionContextMenu.replyInThread',
+        icon: Expensicons.ChatBubble,
+        successTextTranslateKey: '',
+        successIcon: null,
+        shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID) => {
+            return (
+                Permissions.canUseThreads(betas) &&
+                type === CONTEXT_MENU_TYPES.REPORT_ACTION &&
+                reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT &&
+                reportAction.childReportID.toString() !== reportID
```

Comment:
I think this is throwing an error and crashing app.
![image](https://github.com/Expensify/App/assets/14356145/399a86ca-8baf-4fcd-af05-d37f0041b49e)

Reply:
probably cause its an unsafe check as many reportActions don't have childReportID.